pkg/services/ngalert/notifier/channels/pushover.go

```diff
@@ -32,112 +32,90 @@ var (
 // alert notifications to Pushover
 type PushoverNotifier struct {
 	*Base
-	UserKey          string
-	APIToken         string
-	AlertingPriority int
-	OKPriority       int
-	Retry            int
-	Expire           int
-	Device           string
-	AlertingSound    string
-	OKSound          string
-	Upload           bool
-	Message          string
-	tmpl             *template.Template
-	log              log.Logger
-	images           ImageStore
-	ns               notifications.WebhookSender
+	tmpl     *template.Template
+	log      log.Logger
+	images   ImageStore
+	ns       notifications.WebhookSender
+	settings pushoverSettings
```

Comment:
I like this middle ground of using the settings struct pattern, but still staying on simplejson. Nice work :+1: